### ISSUE TITLE
Waterfox crashes randomly browsing facebook or reddit

### ISSUE THREAD

**Issue by x0d3r** [BrowserWorks/waterfox#1135]
**Describe the bug**
On Debian 10 buster, waterfox crashes randomly browsing facebook or reddit

**To Reproduce**
1. Browse reddit or facebook
2. Exception stack:
```shell
	Extension error: Error: Type error for parameter details (Error processing title: Expected string instead of null) for browserAction.setTitle. moz-extension://7f41db0b-6dd2-425c-b407-8e5189cc207c/background.js 716
	[[Exception stack
	updateTimer@moz-extension://7f41db0b-6dd2-425c-b407-8e5189cc207c/background.js:716:4
	handleTabUpdated@moz-extension://7f41db0b-6dd2-425c-b407-8e5189cc207c/background.js:1187:4
	Current stack
	runSafeSyncWithoutClone@resource://gre/modules/ExtensionUtils.jsm:63:129
	runSafeWithoutClone@resource://gre/modules/ExtensionCommon.jsm:174:38
	fire@resource://gre/modules/ExtensionChild.jsm:771:71
	receiveMessage@resource://gre/modules/ExtensionChild.jsm:773:76
	_callHandlers/<@resource://gre/modules/MessageChannel.jsm:587:17
	_callHandlers@resource://gre/modules/MessageChannel.jsm:586:14
	_handleMessage/deferred.promise<@resource://gre/modules/MessageChannel.jsm:651:7
	_handleMessage@resource://gre/modules/MessageChannel.jsm:648:24
	receiveMessage@resource://gre/modules/MessageChannel.jsm:163:5
	]]
	ATTENTION: default value of option force_s3tc_enable overridden by environment.
	[Child 7624] WARNING: pipe error (16): Connection reset by peer: file /home/linuxbox/Waterfox/ipc/chromium/src/chrome/common/ipc_channel_posix.cc, line 353
	[Child 7624] WARNING: pipe error (3): Connection reset by peer: file /home/linuxbox/Waterfox/ipc/chromium/src/chrome/common/ipc_channel_posix.cc, line 353
	Segmentation fault
```

**Desktop:**
 - OS: Debian 10 buster
 - Version: 56.2.14 (64-bit)

**Comment by grahamperrin** [comment]
Please let us have troubleshooting information. 

Use the **Copy text to clipboard** button at: 

- about:support

**Comment by x0d3r** [comment]
```shell

Application Basics
------------------

Name: Waterfox
Version: 56.2.14
Build ID: 20190903144949
Update Channel: release
User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:56.0) Gecko/20100101 Firefox/56.0 Waterfox/56.2.14
OS: Linux 4.19.0-0.bpo.4-amd64
Multiprocess Windows: 1/1 (Enabled by user)
Web Content Processes: 1/1
Stylo: false (disabled by default)
Google Key: Missing
Mozilla Location Service Key: Missing
Safe Mode: false

Waterfox Features
-----------------

Extensions
----------

Name: DownThemAll!
Version: 3.0.8
Enabled: true
ID: {DDC359D1-844A-42a7-9AA1-88A850A938A8}

Name: LeechBlock NG
Version: 1.0.4
Enabled: true
ID: leechblockng@proginosko.com

Name: Session Manager
Version: 0.8.1.13
Enabled: true
ID: {1280606b-2510-4fe0-97ef-9b5a22eafe30}

Graphics
--------

Features
Compositing: Basic
Asynchronous Pan/Zoom: wheel input enabled; scrollbar drag enabled
WebGL 1 Driver WSI Info: GLX 1.4 GLX_VENDOR(client): Mesa Project and SGI GLX_VENDOR(server): SGI Extensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_MESA_swap_control GLX_OML_swap_method GLX_OML_sync_control GLX_SGI_make_current_read GLX_SGI_swap_control GLX_SGI_video_sync GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap GLX_INTEL_swap_event GLX_EXT_buffer_age
WebGL 1 Driver Renderer: Intel Open Source Technology Center -- Mesa DRI Intel(R) Ironlake Desktop
WebGL 1 Driver Version: 2.1 Mesa 13.0.6
WebGL 1 Driver Extensions: GL_ARB_multisample GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_copy_texture GL_EXT_polygon_offset GL_EXT_subtexture GL_EXT_texture_object GL_EXT_vertex_array GL_EXT_compiled_vertex_array GL_EXT_texture GL_EXT_texture3D GL_IBM_rasterpos_clip GL_ARB_point_parameters GL_EXT_draw_range_elements GL_EXT_packed_pixels GL_EXT_point_parameters GL_EXT_rescale_normal GL_EXT_separate_specular_color GL_EXT_texture_edge_clamp GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_ARB_framebuffer_sRGB GL_ARB_multitexture GL_EXT_framebuffer_sRGB GL_IBM_multimode_draw_arrays GL_IBM_texture_mirrored_repeat GL_3DFX_texture_compression_FXT1 GL_ARB_texture_cube_map GL_ARB_texture_env_add GL_ARB_transpose_matrix GL_EXT_blend_func_separate GL_EXT_fog_coord GL_EXT_multi_draw_arrays GL_EXT_secondary_color GL_EXT_texture_env_add GL_EXT_texture_filter_anisotropic GL_EXT_texture_lod_bias GL_INGR_blend_func_separate GL_NV_blend_square GL_NV_light_max_exponent GL_NV_texgen_reflection GL_NV_texture_env_combine4 GL_S3_s3tc GL_SUN_multi_draw_arrays GL_ARB_texture_border_clamp GL_ARB_texture_compression GL_EXT_framebuffer_object GL_EXT_texture_compression_s3tc GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_MESA_window_pos GL_NV_packed_depth_stencil GL_NV_texture_rectangle GL_ARB_depth_texture GL_ARB_occlusion_query GL_ARB_shadow GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_mirrored_repeat GL_ARB_window_pos GL_EXT_stencil_two_side GL_EXT_texture_cube_map GL_NV_depth_clamp GL_APPLE_packed_pixels GL_APPLE_vertex_array_object GL_ARB_draw_buffers GL_ARB_fragment_program GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_vertex_program GL_ARB_vertex_shader GL_ATI_draw_buffers GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_MESA_pack_invert GL_NV_primitive_restart GL_ARB_depth_clamp GL_ARB_fragment_program_shadow GL_ARB_half_float_pixel GL_ARB_occlusion_query2 GL_ARB_point_sprite GL_ARB_shading_language_100 GL_ARB_sync GL_ARB_texture_non_power_of_two GL_ARB_vertex_buffer_object GL_ATI_blend_equation_separate GL_EXT_blend_equation_separate GL_OES_read_format GL_ARB_color_buffer_float GL_ARB_pixel_buffer_object GL_ARB_texture_compression_rgtc GL_ARB_texture_float GL_ARB_texture_rectangle GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_shared_exponent GL_ARB_framebuffer_object GL_EXT_framebuffer_blit GL_EXT_packed_depth_stencil GL_APPLE_object_purgeable GL_ARB_vertex_array_object GL_ATI_separate_stencil GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_gpu_program_parameters GL_EXT_texture_array GL_EXT_texture_integer GL_EXT_texture_sRGB_decode GL_EXT_timer_query GL_OES_EGL_image GL_AMD_performance_monitor GL_ARB_copy_buffer GL_ARB_depth_buffer_float GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_map_buffer_range GL_ARB_texture_rg GL_ARB_texture_swizzle GL_ARB_vertex_array_bgra GL_EXT_texture_swizzle GL_EXT_vertex_array_bgra GL_NV_conditional_render GL_AMD_seamless_cubemap_per_texture GL_ARB_ES2_compatibility GL_ARB_debug_output GL_ARB_draw_elements_base_vertex GL_ARB_explicit_attrib_location GL_ARB_fragment_coord_conventions GL_ARB_provoking_vertex GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_shader_texture_lod GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_vertex_type_2_10_10_10_rev GL_EXT_provoking_vertex GL_EXT_texture_snorm GL_MESA_texture_signed_rgba GL_NV_texture_barrier GL_ARB_get_program_binary GL_ARB_robustness GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_compressed_texture_pixel_storage GL_ARB_internalformat_query GL_ARB_map_buffer_alignment GL_ARB_shading_language_packing GL_ARB_texture_storage GL_AMD_shader_trinary_minmax GL_ARB_arrays_of_arrays GL_ARB_clear_buffer_object GL_ARB_copy_image GL_ARB_explicit_uniform_location GL_ARB_invalidate_subdata GL_ARB_program_interface_query GL_ARB_vertex_attrib_binding GL_KHR_debug GL_KHR_robustness GL_ARB_buffer_storage GL_ARB_clear_texture GL_ARB_internalformat_query2 GL_ARB_multi_bind GL_ARB_seamless_cubemap_per_texture GL_ARB_shader_draw_parameters GL_ARB_texture_mirror_clamp_to_edge GL_ARB_vertex_type_10f_11f_11f_rev GL_INTEL_performance_query GL_ARB_clip_control GL_ARB_get_texture_sub_image GL_ARB_pipeline_statistics_query GL_ARB_texture_barrier GL_EXT_polygon_offset_clamp GL_KHR_blend_equation_advanced GL_KHR_context_flush_control
WebGL 1 Extensions: ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_frag_depth EXT_sRGB EXT_shader_texture_lod EXT_texture_filter_anisotropic MOZ_debug OES_element_index_uint OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context MOZ_WEBGL_lose_context MOZ_WEBGL_compressed_texture_s3tc MOZ_WEBGL_depth_texture
WebGL 2 Driver WSI Info: -
WebGL 2 Driver Renderer: WebGL creation failed: * Error during native OpenGL init. * Error during native OpenGL init. * Error during native OpenGL init. * Error during native OpenGL init. * Error during native OpenGL init. * Exhausted GL driver caps. * Exhausted GL driver options.
WebGL 2 Driver Version: -
WebGL 2 Driver Extensions: -
WebGL 2 Extensions: -
GPU #1
Active: Yes
Description: Intel Open Source Technology Center -- Mesa DRI Intel(R) Ironlake Desktop
Vendor ID: Intel Open Source Technology Center
Device ID: Mesa DRI Intel(R) Ironlake Desktop
Driver Version: 2.1 Mesa 13.0.6

Diagnostics
AzureCanvasAccelerated: 0
AzureCanvasBackend: skia
AzureContentBackend: skia
AzureFallbackCanvasBackend: none
CairoUseXRender: 0
Decision Log
HW_COMPOSITING:
blocked by default: Acceleration blocked by platform
OPENGL_COMPOSITING:
unavailable by default: Hardware compositing is disabled
WEBRENDER:
opt-in by default: WebRender is an opt-in feature
unavailable by runtime: Build doesn't include WebRender




Media
-----

Audio Backend: pulse
Max Channels: 2
Preferred Channel Layout: stereo
Preferred Sample Rate: 44100
Output Devices
Name: Group
Built-in Audio Analog Stereo: /devices/pci0000:00/0000:00:1b.0/sound/card0
Input Devices
Name: Group
Monitor of Built-in Audio Analog Stereo: /devices/pci0000:00/0000:00:1b.0/sound/card0

Important Modified Preferences
------------------------------

accessibility.typeaheadfind.flashBar: 0
browser.cache.disk.capacity: 358400
browser.cache.disk.filesystem_reported: 1
browser.cache.disk.hashstats_reported: 1
browser.cache.disk.smart_size.first_run: false
browser.cache.disk.smart_size.use_old_max: false
browser.cache.frecency_experiment: 2
browser.places.smartBookmarksVersion: 8
browser.sessionstore.upgradeBackup.latestBuildID: MOZ_BUILDID
browser.startup.homepage_override.buildID: MOZ_BUILDID
browser.startup.homepage_override.mstone: 56.2.14
browser.tabs.warnOnClose: false
browser.urlbar.lastSuggestionsPromptDate: 20180715
browser.urlbar.timesBeforeHidingSuggestionsHint: 0
dom.disable_open_during_load: false
extensions.lastAppVersion: 56.2.14
font.internaluseonly.changed: false
media.eme.enabled: true
media.gmp-gmpopenh264.abi: x86_64-gcc3
media.gmp-gmpopenh264.lastUpdate: 1566492417
media.gmp-gmpopenh264.version: 1.8.1
media.gmp-manager.buildID: MOZ_BUILDID
media.gmp-manager.lastCheck: 1568220620
media.gmp-widevinecdm.abi: x86_64-gcc3
media.gmp-widevinecdm.lastUpdate: 1566492423
media.gmp-widevinecdm.version: 4.10.1440.19
media.gmp.storage.version.observed: 1
network.cookie.prefsMigrated: true
network.predictor.cleaned-up: true
places.database.lastMaintenance: 1567445303
places.history.expiration.transient_current_max_pages: 122641
plugin.disable_full_page_plugin_for_types: application/pdf
privacy.cpd.extensions-sessionmanager: false
services.sync.declinedEngines:
storage.vacuum.last.index: 1
storage.vacuum.last.places.sqlite: 1567445300
storage.vacuum.last.queue.sqlite: 1565801877

Important Locked Preferences
----------------------------

Places Database
---------------

JavaScript
----------

Incremental GC: true

Accessibility
-------------

Activated: false
Prevent Accessibility: 0

Library Versions
----------------

NSPR
Expected minimum version: 4.16
Version in use: 4.16

NSS
Expected minimum version: 3.34
Version in use: 3.34

NSSSMIME
Expected minimum version: 3.34
Version in use: 3.34

NSSSSL
Expected minimum version: 3.34
Version in use: 3.34

NSSUTIL
Expected minimum version: 3.34
Version in use: 3.34

Experimental Features
---------------------

Sandbox
-------

Seccomp-BPF (System Call Filtering): true
Seccomp Thread Synchronization: true
User Namespaces for privileged processes: true
User Namespaces: false
Content Process Sandboxing: true
Media Plugin Sandboxing: true
Content Process Sandbox Level: 1
Effective Content Process Sandbox Level: 1

Rejected System Calls
---------------------

```

**Comment by hawkeye116477** [comment]
Related => https://github.com/proginosko/LeechBlockNG/issues/86
So it's problem with LeechBlock NG extension, downgrade to 0.9.7 and maybe that will fix problem.
